src/ServiceControl/Infrastructure/RavenDB/RavenBootstrapper.cs

```diff
@@ -74,11 +74,12 @@ public void StartRaven(EmbeddableDocumentStore documentStore, Settings settings,
 
             documentStore.Configuration.DisableClusterDiscovery = true;
             documentStore.Configuration.ResetIndexOnUncleanShutdown = true;
-            documentStore.Configuration.Port = settings.Port;
+            documentStore.Configuration.Port = settings.MaintenancePort;
             documentStore.Configuration.HostName = settings.Hostname == "*" || settings.Hostname == "+"
                 ? "localhost"
                 : settings.Hostname;
-            documentStore.Configuration.VirtualDirectory = $"{settings.VirtualDirectory}/storage";
+            // TODO: Still required?
```

Comment:
I'm on the fence about this. The assumption previously was that it is possible to have a virtual directory but it seems to not be fully implemented plus now with the split would we need to introduce a dedicated virtual directory for the maintenance mode since we are no longer sharing the same base URL?

Reply:
I'd drop the virtual directory